third_party/rust/kuchikiki/v0_8/crate/Cargo.toml

```diff
@@ -31,10 +31,11 @@ doctest = false
 version = "0.27"
 
 [dependencies.html5ever]
-version = "0.26.0"
+version = "0.25.1"
 
 [dependencies.indexmap]
-version = "1.6.0"
+version = "1.9.3"
+features = ["std"]
```

Comment:
can you remove this comment? It doesn't seem relevant now and would be overwritten with autogenerated config anyway https://github.com/brave/brave-core/blob/master/third_party/rust/indexmap/v1/BUILD.gn#L65
https://github.com/brave/brave-core/blob/master/third_party/rust/indexmap/v1/BUILD.gn#L65

Reply:
Fixed in 063aa23bde77a5579db8352c2aa972d190668b9d.